src/backend/distributed/commands/index.c

```diff
@@ -306,37 +306,25 @@ ExecuteFunctionOnEachTableIndex(Oid relationId, PGIndexProcessor pgIndexProcesso
 								int indexFlags)
 {
 	List *result = NIL;
-	ScanKeyData scanKey[1];
-	int scanKeyCount = 1;
 
-	PushOverrideEmptySearchPath(CurrentMemoryContext);
```

Comment:
I'm wondering whether this line was here for a reason. Usually we do this such that the deparser always generates fully-qualified names, but I'm not sure we actually call deparsing logic from pgIndexProcessor.

Reply:
We probably do in GatherIndexAndConstraintDefinitionList, though it seems preferable to do the PushOverrideEmptySearchPath there.

Reply:
Looks like I should keep them in case a further user of this infra wants to generate fully-qualified names.

Reply:
> We probably do in GatherIndexAndConstraintDefinitionList, though it seems preferable to do the PushOverrideEmptySearchPath there.

I've seen that later, that makes more sense 👍

Reply:
or maybe we should do that in `GetTableIndexAndConstraintCommands` & `GetTableIndexAndConstraintCommandsExcludingReplicaIdentity`